docs/elements.md

```diff
@@ -2317,10 +2317,10 @@ Displays the contents of question directions.
 
 #### Details
 
-Contents are only shown during question input portion. When a student
-either makes a submission or receives the correct answer, the information
-between these tags is hidden. If content exists outside of a question panel,
-then it will be displayed alongside the answer.
+Generally, any text placed in `question.html` will appear in 
```

Comment:
Can this description be copied to the other panel elements?

Reply:
If you were to spend time on this, I think @nwalters512's suggestion is probably most appropriate:

> In a perfect (and hopefully future) world, we'd have documentation that explains panels, and then we can just have the docs for all these elements be something like this:

> > Ensures that content is only displayed in the question panel. See (link to panel documentation) for more details).

Reply:
Yeahhh, probably not worth fussing over and just waiting for the docs revamp.

Reply:
> To be clear, my (firm) suggestion was to make the same changes to the other elements _now_, and save the part where we centralize that explanation for some hypothetical future revamp.

That makes sense, I had the same thing in mind when making my original comment. I'll go ahead and copy everything to make things consistent.

Reply:
@nwalters512 changes made here https://github.com/PrairieLearn/PrairieLearn/pull/8170/commits/1c612d73e6ce5d51522a25a1a5349e4ae3e9dfa6